person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name: 'Unknown', age: nil, parent_permission: true)
```

Comment:
- [ ] It would be nice if you make the age a mandatory parameter because it is clearly mentioned in the requirements that name and parent_permission are optional parameter 👍 So your code should like below ⬇️

```ruby
def initialize(age, name = "Unknown", parent_permission = true)
```